### DISCUSSION TITLE
Can't connection to server on '127.0.0.1' (10061)

### DISCUSSION THREAD

**Discussion by mhmmdtech** [leokhoa/laragon#511] [Q&A] [answered]
Hello, I hope you are well
I have installed Laragon and I get this error mostly after I turn it on and try to connect to my MySQL through HeidiSQL.
Do you know what the solution is?

**Comment by Pen-y-Fan** [comment]
There could be several reasons, see my blog [How to set up Laragon on a new Windows computer (part 1) - Install Laragon](https://pen-y-fan.github.io/2022/11/07/how-to-set-up-laragon-on-a-new-windows-computer/), in particular the part on permissions.

**Reply by mhmmdtech** [reply]
I read your article, It was very good, thank you, but it did not help at all to solve the problem.

**Comment by chmst** [comment]
If you are on win11: shutdown and re-start helped for me.

**Comment by istiyakamin** [comment, marked as answer]
This error (10061) usually means the server isn't running or is blocked by a firewall. Make sure Laragon is fully started and check your firewall settings to allow connections on port 80 (or whatever port you're using). Sometimes, another application might be using the same port, so you could try changing Laragon's port in its settings. Also, ensure your hosts file (C:\Windows\System32\drivers\etc\hosts) doesn't have any conflicting entries for 127.0.0.1.

**Comment by nnay29** [comment]
For me it was my Antivirus that blocked.
I authorized laragon as safe software and the issue was solved.
